.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
I think this belongs to user's .gitignore
`git config --global core.excludesfile $HOME/.gitignore`

Reply:
done 👍